recipe/meta.yaml

```
@@ -23,14 +23,14 @@ requirements:
     - pip
     - hatchling >=1.11
     - hatch-jupyter-builder >=0.5
-    - jupyterlab >=4.4.4,<4.5
+    - jupyterlab >=4.5.0rc0,<4.6
```

It's a bit of a grey area but we (Anaconda) generally don't publish release candidate builds so the recipe probably shouldn't refer to it.

```suggestion
    - jupyterlab >=4.5.0,<4.6
```